src/TextInputCustom.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef, ReactElement } from 'react';
+import { GestureResponderEvent, TextInput, TextInputProps } from 'react-native';
+
+// eslint-disable-next-line @typescript-eslint/interface-name-prefix
+export interface ITextInputCustomProps extends TextInputProps {
+  ref: any;
+  onPress?: (event: GestureResponderEvent) => void;
+}
+
+// eslint-disable-next-line react/display-name
+const TextInputCustom: React.ForwardRefExoticComponent<React.PropsWithoutRef<ITextInputCustomProps> & React.RefAttributes<unknown>> = forwardRef(
+  (props: ITextInputCustomProps, ref): ReactElement => {
+    const { onPress, editable } = props;
+    // This hack needs to get click position and then calculate what cell on clicked
+    const _onPress = (event: GestureResponderEvent) => {
+      if (onPress && (editable || editable === undefined)) {
+        onPress(event);
+      }
+    };
+    return <TextInput ref={ref} {...props} editable={editable} onTouchStart={_onPress} />;
```

Comment:
onTouchStart never invoked on Android  RN@0.62

Reply:
Yes it’s there for compatibility

Reply:
I want to say that is a problem that 

> onTouchStart never invoked on Android RN@0.62